tests/sftp/hooks/test_sftp.py

```diff
@@ -20,6 +23,12 @@ async def listdir(self, path: str):
         else:
             return ["..", ".", "file"]
 
+    async def readdir(self, path: str):
```

Comment:
```suggestion
    async def read_dir(self, path: str):
```

Reply:
We're mocking the `readdir` method of the `asyncssh.sftp` client. Since this method has been declared this way, we have to mock it that way.